### PR TITLE
Refactor/fund op flow

### PR DESCRIPTION
**Pull request by valentinnavalos**

### CODE REVIEW

File: packages/web-app/src/containers/actionBuilder/fundOpportunity/index.tsx [hunk 1]
```diff
@@ -1,30 +1,53 @@
...
+import { ActionIndex, CollateralType, FundingSource } from "utils/types";
 
 const fundingSourceOptions: {
     label: string;
     value: FundingSource;
 }[] = [
-    { label: 'DAO treasury', value: 'DAO' },
-    { label: 'Aave borrow', value: 'AAVE' },
-];
+        { label: 'DAO treasury', value: 'DAO' },
```
Comment (ialberquilla) [inline packages/web-app/src/containers/actionBuilder/fundOpportunity/index.tsx:12]: Better as a type, we will need then use these fields in the action encoding

File: packages/web-app/src/containers/actionBuilder/fundOpportunity/index.tsx [hunk 2]
```diff
@@ -34,11 +57,14 @@ const FundOpportunityStepOne: React.FC = () => {
...
+                    name={`actions.${actionIndex}.inputs.fundingSource`}
                     control={control}
                     render={({ field: { onChange, value, name } }) => (
-                        <StyledSelect {...{ name, value, onChange }}>
-                            <option value="" disabled selected>{t('creditDelegation.selectAnOption')}</option>
+                        <StyledSelect
+                            {...{ name, value, onChange }}
+                            defaultValue={""}
+                        >
+                            <option value="" disabled>{t('creditDelegation.selectAnOption')}</option>
```
Comment (ialberquilla) [inline packages/web-app/src/containers/actionBuilder/fundOpportunity/index.tsx]: credit delegation?

File: packages/web-app/src/containers/fundOpportunityStepper/index.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                        disableAddAction
+                        hideAlert
+                        allowEmpty={false}
+                    />
+                </Step>
+                <Step isNextButtonDisabled={!stepTwoIsValid}>
+                    <FundOpportunityStepTwo />
+                </Step>
+                <Step
+                    wizardTitle={t('creditDelegation.defineProposal.heading')}
```
Comment (ialberquilla) [inline packages/web-app/src/containers/fundOpportunityStepper/index.tsx]: References to credit delegation

File: packages/web-app/src/containers/actionBuilder/fundOpportunity/index.tsx (hunk 1 above)
Comment (valentinnavalos) [reply packages/web-app/src/containers/actionBuilder/fundOpportunity/index.tsx:12]: I do not understand. `fundingSourceOptions.value` variable already has a 'type'. What are you referring to?

Comment (ialberquilla) [reply packages/web-app/src/containers/actionBuilder/fundOpportunity/index.tsx:12]: OK, yes my bad, the type is defined in another file

**Pull request merged**